framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceJavaParserStorage.java

```diff
@@ -1039,7 +1042,7 @@ public void writeResultsToFile(OutputFormat outputFormat, BaseTypeChecker checke
     for (String path : modifiedFiles) {
       // This calls deepCopy() because wpiPrepareCompilationUnitForWriting performs side effects
       // that we don't want to be persistent.
-      CompilationUnitAnnos root = sourceToAnnos.get(path).deepCopy();
+      CompilationUnitAnnos root = sourceToAnnos.get(path);
```

Comment:
Why did you make this change?  Also, it is now inconsistent with the comment immediately above.

Reply:
This change was presumably made to avoid the bug fixed by #5884, and we should undo it once that PR is merged.